src/init/features/dataconnect/index.ts

```diff
@@ -113,7 +113,7 @@ export async function doSetup(setup: Setup, config: Config): Promise<void> {
     cloudSqlDatabase = await promptOnce({
       message: `What ID would you like to use for your new database in ${cloudSqlInstanceId}?`,
       type: "input",
-      default: `dataconnect`,
+      default: `postgres`,
```

Comment:
Is the goal to improve database in emulator?

This changes the prod Cloud SQL database name, which I think a separate name it more clear. I agree that we should make the local emulator url as default as possible.

https://stackoverflow.com/questions/3582552/what-is-the-format-for-the-postgresql-connection-string-url

Reply:
Let's continue the internal discussion on b/339903547 please